locales/en/app.ftl

```diff
@@ -38,6 +38,7 @@ user-add-invalid-email = Invalid Email
 user-add-email-verify-subject = Verify your subscription to {-product-name}.
 user-add-title = {-product-name} : Confirm Email
 
+error = Error
```

Comment:
Is this a title? Maybe it would be nice to have a different ID, e.g. error-title, to make the context clearer and avoid the temptation to reuse it in a different context